src/class-config.php

```diff
@@ -378,6 +378,23 @@ protected function get_acf_field_value( $root, $acf_field, $format = false ) {
 			$id = $root['node']->ID;
 		}
 
+		if ( is_array( $root ) && isset( $root[ $acf_field['name'] ] ) ) {
+			return $root[ $acf_field['name'] ];
+		}
+
+		// If the field is on a block, try and resolve from the block fields
+		if ( is_array( $root ) && isset( $root['blockName'], $root['attrs']['data'][ $acf_field['name'] ] ) && is_array( $root ) ) {
+
+			// @see: https://support.advancedcustomfields.com/forums/topic/getting-get_field-outside-block-loop/#post-84022
+			$block_id = isset( $root['attrs']['id'] ) ? $root['attrs']['id'] : null;
+			acf_setup_meta( $root['attrs']['data'], $block_id, true );
```

Comment:
@lgladdy hmm. Making that change breaks for me. 

This is something that will only work on v6?

Reply:
Shouldn't be... that logic hasn't been change. I'll test further!

Reply:
Turns out, `acf_setup_meta` doesn't build and return the right structure for repeater (and repeater like) fields, so that's why it's erroring.

Instead of that route, changing from a `null` block_id to a random `block_{random-id}` will solve the issue for ACF6.